components/region_cache_memory_engine/src/engine.rs

```diff
@@ -813,22 +874,23 @@ mod tests {
 
     #[test]
     fn test_get_value() {
-        let engine = RegionCacheMemoryEngine::default();
-        engine.new_region(1);
+        let engine = RangeCacheMemoryEngine::new(Arc::default());
+        let range = CacheRange::new(b"k000".to_vec(), b"k100".to_vec());
```

Comment:
are we testing the edge condition where one of the cached ranges is open one end or the other? e.g., from first data key, to the end?